SPEC.md

```diff
@@ -130,3 +130,11 @@ The test index makes use of symbolic language in describing connection and messa
 
     The node should *NOT* request mtGET_PEER_SHARD_INFO_V2 after connection if there was no handshake.
     The test waits for the predefined amount of time, ensuring no such messages were received.
+
+### ZG-CONFORMANCE-008
+    
+    The node should respond with transaction details after receiveing TmGetObjectByHash / OtTransactions request.
```

Comment:
It might be good to make a note of the rpc intricacies here, since it's necessary for this case?